### PR TITLE
feat(GKEE): add namespace memory dashboard

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json [hunk 1]
```diff
@@ -0,0 +1,123 @@
+{
+  "category": "CUSTOM",
+  "displayName": "GKE Enterprise Namespace Observability Memory",
+  "dashboardFilters": [],
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "widget": {
+          "title": " Memory Request % Used (Top 5 Namespaces)",
```
Comment (yqlu) [inline dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json:10]: nit: remove leading space

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            }
+          }
+        },
+        "height": 16,
+        "width": 24,
+        "yPos": 16
+      },
+      {
+        "widget": {
+          "title": " Requested Memory Unused (Top 5 Namespaces)",
```
Comment (yqlu) [inline dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json:93]: Nit: remove leading space

**Review by yqlu — APPROVED** [review]
Dashboard itself looks great! Just a formatting nitpick.

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json (hunk 1 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json]: Thanks - Fixed!

File: dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json (hunk 2 above)
Comment (apeabody) [reply dashboards/google-kubernetes-engine-enterprise/gke-enterprise-namespace-observability-memory.json]: Thanks - Fixed!

**Pull request merged**